etc/entrypoint.sh

```diff
@@ -1,4 +1,15 @@
-#!/usr/bin/env sh
```

Comment:
depend on target shell and yours is unknown on shell script
ref: https://github.com/prest/prest/issues/449

Reply:
busybox shell do not parse arguments so I change to #!/bin/sh